docs/en/Configuring-Alluxio-with-secure-HDFS.md

```diff
@@ -62,6 +62,14 @@ Then edit `alluxio-site.properties` file to set the under storage address to the
 Copy secure HDFS conf xml files (`core-site.xml`, `hdfs-site.xml`, `mapred-site.xml`, `yarn-site.xml`) to
 `${ALLUXIO_HOME}/conf/`
 
+Optionally, you can set jvm-level system properties for customized Kerberos configurations:
+`java.security.krb5.realm` and `java.security.krb5.kdc`. Those Kerberos configurations route java libraries to specified Kerberos realm and KDC server address.
+If both are set to empty, Kerberos library will respect
+the default Kerberos configuration on the machine. For example, if you use Hadoop or Spark, you can set those two configurations in `HADOOP_OPTS` or `SPARK_OPTS` with `-D`.
```

Comment:
I'm not sure if this is specific enough, ie which file would they set these environment (?) variables?